route/target.go

```diff
@@ -32,4 +32,7 @@ type Target struct {
 
 	// timerName is the name of the timer in the metrics registry
 	timerName string
+
```

Comment:
"TLSSkipVerify disables certificate validation for upstream TLS connections."

Could you also move the variable below the `StripPath` variable, please?